builds.go

```diff
@@ -4,22 +4,26 @@ import (
 	"context"
 	"crypto/sha256"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"os"
 	"os/exec"
 	"strings"
 	"sync"
+	"time"
 
 	"github.com/docker-library/meta-scripts/om"
 
 	c8derrdefs "github.com/containerd/containerd/errdefs"
+	c8derrs "github.com/containerd/containerd/remotes/errors"
 	"github.com/docker-library/bashbrew/registry"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/sirupsen/logrus" // this is used by containerd libraries, so we need to set the default log level for it
+	"golang.org/x/time/rate"
 )
 
-var concurrency = 50
+var concurrency = 1000
```

Comment:
How does this interact with the rate limiter?

Reply:
This was implemented initially as a poor-man's rate limiter, so now that we have a "real" rate limiter in place, we could technically remove this completely and let it be completely unbounded, but I figured making it high (instead of unlimited) was probably better/safer.

In practice the rate limiter is likely to block many of these goroutines anyhow (and having them wait to make a request vs wait to start seems totally fair since the overhead is really low and they can potentially do other things in the mean time).  Also, with the cache implemented, many of the goroutines won't block at all, so it's just "free" concurrency. :+1: